tests/queries/0_stateless/01275_parallel_mv.reference

```diff
@@ -1,4 +1,27 @@
+-- { echoOn }
+set parallel_view_processing=1;
+insert into testX select number from numbers(10) settings log_queries=1; -- { serverError FUNCTION_THROW_IF_VALUE_IS_NON_ZERO }
+system flush logs;
+select length(thread_ids) from system.query_log where current_database = currentDatabase() and type != 'QueryStart' and query like '%insert into testX %' and Settings['parallel_view_processing'] = '1';
+8
+select count() from testX;
 10
+select count() from testXA;
 10
+select count() from testXB;
 0
+select count() from testXC;
 10
+set parallel_view_processing=0;
```

Comment:
Note, that now (I guess after #28582) with `parallel_view_processing=0` MVs that goes after MV with an error will be processed too.

Reply:
That was changed in 21.9 so it was deterministic and all views were processed fully; otherwise they could be aborted after any of the 3 steps (prefix, write, suffix) and it was impossible to know what was the status of the target table. So no (new) behaviour change.

Reply:
Ok, got it, thanks!